operator/api/v1alpha1/central_types.go

```diff
@@ -602,6 +606,10 @@ const (
 	ScannerV4ComponentDisabled ScannerV4ComponentPolicy = "Disabled"
 )
 
+type ConfigAsCodeSpec struct {
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
Please change this to an enum, we're [trying to avoid booleans](https://github.com/stackrox/stackrox/blob/master/operator/EXTENDING_CRDS.md#data-type-choices).